### PR TITLE
#115231329 create custom 404 page for project

### PR DESCRIPTION
**Pull request by collinmutembei**
#### What does this PR do?

Add custom `404` page for the project, to be rendered when there is a request to a route that does not exist. The page also allows for users to navigate back to the projects' homepage.
#### Description of Task to be completed?

Create a template to display a `404` error message for urls that do not exist on the site or urls users are not meant to view.
#### How should this be manually tested?

Navigating to a meaningless url on the site, e.g http://dev.limberapp.xyz/over9000 displays the `404` page.
#### Any background context you want to provide?

When a user navigated to a url that did not exist, the Django server would display the default `404` page with a list of all allowed urls as a way to help with debugging. This could lead to exposure of the internal workings of **Limber**. To alleviate such incidents, having a custom `404` template for the project will be displayed instead.

It is worth noting that setting `DEBUG` setting to false makes Django server stop serving up static files especially on localhost. To allow for that to happen, it is necessary to run the Django server with the `insecure` directive as indicated [here](http://stackoverflow.com/questions/5836674/why-does-debug-false-setting-make-my-django-static-files-access-fail).

```
$ python manage.py runserver --insecure
```
#### What are the relevant pivotal tracker stories?

The story corresponding to this `PR` is [#115231329](https://www.pivotaltracker.com/story/show/115231329)
#### Screenshot

image

### CODE REVIEW

File: public/static/styles/404.css [hunk 1]
```diff
@@ -0,0 +1,949 @@
...
+
+.col-1 {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 89px;
+  height: 183px;
+  background: #ffc928;
+}
+.col-1 .shadow {
```
Comment (andela-anandaa) [inline public/static/styles/404.css:40]: Skip a line here.

File: public/static/styles/404.css [hunk 2]
```diff
@@ -0,0 +1,949 @@
...
+}
+.col-1 .inner-box .border-4-4 {
+  border-radius: 4px;
+  background: black;
+  height: 10px;
+  width: 68px;
+  position: absolute;
+  top: 183px;
+  left: 0px;
+}
```
Comment (andela-anandaa) [inline public/static/styles/404.css:142]: Same to the rest, a line should be skipped between _selectors_.

File: public/static/styles/404.css [hunk 3]
```diff
@@ -0,0 +1,949 @@
...
+  position: absolute;
+  top: -5px;
+  left: 10px;
+  border-top: 0px;
+  border-right: 0px;
+  border-bottom: 0px;
+  border-radius: 50% 0px 0px 0px;
+  z-index: 10;
+}
+.pacman-container .pacman-body .tail {
```
Comment (andela-anandaa) [inline public/static/styles/404.css:421]: Write comments for the CSS animation of _Pacman_.

**Comment by collinmutembei** [comment]
@andela-anandaa I have attended to the fixes highlighted

**Pull request merged**